admin_manual/appliance/firewall.rst

```diff
@@ -0,0 +1,18 @@
+Firewall protected environment
+------------------------------
+
+If you are considering to setup the appliance in an environment with a firewall, 
+please create rules that permit acces to the following hosts. 
+If you DNS is not working, you can take the IP addresses instead.
+If you are using google as your DNS server, you have to permitt acces to it too.
+
+Firewall Rules::
+
+	software-univention.de
+	owncloud.org
+	owncloud.com
+	docker.software-univention.de
+	176.9.114.147
+	marketplace.owncloud.com
+	5.9.68.237
+  8.8.8.8
```

Comment:
indent missing
and could mention above somewhere, that ``8.8.8.8`` is the Google DNS address.